frontend/src/metabase/components/FieldValuesWidget/utils.ts

```diff
@@ -174,22 +176,6 @@ function getSearchableTokenFieldPlaceholder(
   return placeholder;
 }
 
-export function hasList({
-  parameter,
-  fields,
-  disableSearch,
-  options,
-}: {
-  parameter?: Parameter;
-  fields: Field[];
-  disableSearch: boolean;
-  options: FieldValue[];
-}) {
-  return (
-    shouldList({ parameter, fields, disableSearch }) && !_.isEmpty(options)
-  );
-}
```

Comment:
The goal of this commit was to remove this `hasList` function, which was unnecessary indirection and further confused things